openstack/tools/charmed_openstack_functest_runner.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -ux
+source ~/novarc
+CIDR=`openstack subnet show subnet_${OS_USERNAME}-psd-extra -c cidr -f value`
+FIP_MAX=$(ipcalc $CIDR| awk '$1=="HostMax:" {print $2}')
+FIP_MIN=$(ipcalc $CIDR| awk '$1=="HostMin:" {print $2}')
+FIP_MIN_ABC=${FIP_MIN%.*}
+FIP_MIN_D=${FIP_MIN##*.}
+FIP_MIN=${FIP_MIN_ABC}.$(($FIP_MIN_D + 64))
+
+# More information on confog https://github.com/openstack-charmers/zaza/blob/master/doc/source/runningcharmtests.rst
+export TEST_HTTP_PROXY=http://squid.internal:3128
+export TEST_FIP_RANGE=FIP_MIN:FIP_MAX
+export TEST_CIDR_EXT=CIDR
+export TEST_GATEWAY=$(openstack subnet show subnet_${OS_USERNAME}-psd-extra -c gateway_ip -f value)
+export TEST_NAME_SERVER=91.189.91.131
+export TEST_CIDR_PRIV=192.168.21.0/24
+export TEST_SWIFT_IP=10.140.56.22
+export TEST_MODEL_SETTINGS="image-stream=released;default-series=jammy;test-mode=true;transmit-vendor-metrics=false"
+export TEST_CONSTRAINTS_FILE=https://raw.githubusercontent.com/openstack-charmers/zaza/master/constraints-juju34.txt
+# We need to set TEST_JUJU3 as well as the constraints file
+# Ref: https://github.com/openstack-charmers/zaza/blob/e96ab098f00951079fccb34bc38d4ae6ebb38606/setup.py#L47
+export TEST_JUJU3=1
+export NET_ID=$(openstack network show net_${OS_USERNAME}-psd -f value -c id)
```

Comment:
Just making sure this wasn't supposed to be -psd-extra network

Reply:
ah well spotted, this is the "external" network used for fips so for us thats \*-psd-extra